### ISSUE TITLE
Type bug in the resolvers tutorial example page

### ISSUE THREAD

**Issue by nicosefer** [apollographql/apollo#532]
There's a type error on the `missionPatch` resolver. The first argument should be a `Mission` type instead of a `String` type

```
type Mission {
    # ... with rest of schema
    missionPatch(mission: String, size: PatchSize): String
  }
```

**Comment by jcace** [comment]
Hey. New to Apollo here. I don't really understand why we need this `mission` argument in the schema at all? 

Why can't it just be 
```javascript
missionPatch(size: PatchSize): String
```
?

When I take it out, everything still seems to work okay.  Correct me if I'm wrong, but doesn't this `mission` argument mean that when someone's querying missionPatch, they can specify a mission (which doesn't make sense to me... and in the resolver, we're not even destructuring this second argument) 

This is the kind of query I'd imagine this schema supports: passing a separate mission into the missionPatch query (but then the missionPatch wouldn't match the mission that's being returned from the query?)

```
query MissionPatch {
  launch(id: 70){
    mission {
      name
      missionPatch(mission: "Telstar 18V", size: SMALL)
    }
  }
}
```

Thoughts? Thanks!

Jason

**Comment by pcattori** [comment]
Fixed by #879

**Comment by StephenBarlow** [comment]
Indeed it was! Closing this issue.
